frontend/src/modules/teamcomments/components/TeamComments.js

```diff
@@ -0,0 +1,60 @@
+/* @flow */
+
+import * as React from 'react';
+import { Localized } from '@fluent/react';
+
+import './TeamComments.css';
+
+import { AddComment, CommentsList } from 'core/comments';
+
+import type { UserState } from 'core/user';
+import type { TeamCommentState } from 'modules/teamcomments';
+
+type Props = {|
+    teamComments: TeamCommentState,
+    user: UserState,
+    addComment: (string, ?number) => void,
+|};
+
+
+export default function TeamComments(props: Props) {
+    const { teamComments, user, addComment } = props;
+
+    let canComment = user.isAuthenticated;
+    const comments = teamComments.comments;
+
+    if (!comments) {
+        return null;
+    }
+
+    if (!comments.length) {
+        return <section className="team-comments">
+            { !canComment ?
+                <div className="no-team-comments">
+                    <Localized id="entitydetails-Helpers--no-comments">
+                        <p>No comments available.</p>
+                    </Localized>
+                </div>
+                :
+                <div className='comments-list'>
```

Comment:
This case is already handled by `CommentsList`. Simply pass an empty list of comments and it will display the form only. You thus should not be repeating this code here, and count on `CommentsList` to display the form when there are no comments.